TUM Campus App/Grades.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  File.swift
+//  Campus
+//
+//  Created by Tim Gymnich on 20.10.17.
+//  Copyright © 2017 LS1 TUM. All rights reserved.
+//
+
+import Foundation
+//
+//  Grade.swift
+//  TUM Campus App
+//
+//  Created by Florian Gareis on 04.02.17.
+//  Copyright © 2017 LS1 TUM. All rights reserved.
+//
+
+import Foundation
+
+
+class Grades: DataElement {
+    
+    let name: String
```

Comment:
I like the new Model Classes. These will actually go very well with the new refactored network layer. So I would probably put this off until we're ready to have both at the same time

Reply:
Sounds great.